controllers/AuthController.php

```diff
@@ -32,17 +32,29 @@ public function behaviors()
                 'class' => AccessControl::class,
                 'rules' => [
                     [
-                        'actions' => ['signup', 'login', 'request-password-reset', 'reset-password', 'auth'],
+                        'actions' => [
+                            'request-password-reset',
```

Comment:
why allow password reset if already logged in?

Reply:
Yes. Same as it was before the change. That's fine because you may want to change your password even if you're currently logged in.